api/urls.py

```diff
@@ -43,7 +44,7 @@
 router.register(r'course-admin', CourseAdminViewSet, basename='admin-course')
 router.register(r'uqj-update', UpdateUQJViewSet, basename='uqj-update')
 router.register(r'question-report', QuestionReportViewSet, basename='question-report')
-router.register(r'analytics', AnalyticsViewSet, basename='analytics')
+router.register(r'submission-analytics', AnalyticsViewSet, basename='submission-analytics')
```

Comment:
don't change this. the paths for the apis should be like this. `/analytics/submission` or `/analytics/question`

Reply:
Fixed.